larrybuckalew/heart-lab-ai
# Fix Build Issues and Add Root Layout

Fixes build issues and adds necessary configuration:

1. Added root layout with proper metadata
2. Added globals.css with TailwindCSS configuration
3. Updated render.yaml for deployment
4. Updated next.config.js for production build

To implement:
1. Pull latest changes
2. Run npm install
3. Run npm run dev to test locally
4. Deploy to see fixes in production
